firmware/components/driver_framebuffer/Kconfig

```diff
@@ -1,4 +1,4 @@
-menu "Driver: framebuffer support"
+menu "Driver: framebuffer & matrices support"
```

Comment:
These matrices are not separate from the framebuffer driver, are they? It is my understanding that they are used to transform render layers ("windows"). I see them more as animation properties on windows than as a separate functionality.